test/common/storagelib.py

```diff
@@ -83,13 +83,15 @@ def add_loopback_disk(self, size=50, name=None):
         dev = self.machine.execute("F=$(mktemp /var/tmp/loop.XXXX); "
                                    "truncate --size=%sMB $F; "
                                    "losetup --show %s $F" % (size, name if name else "--find")).strip()
+        backfile = self.machine.execute(f"losetup -n -O BACK-FILE -l {dev}")
```

Comment:
Let's please avoid the roundtrip here. Instead, do `backfile = self.machine.execute("mktemp...")` instead of the `F=`, and use f-strings to call losetup with `backfile`.